desktop/libs/indexer/src/indexer/indexers/sql_tests.py

```diff
@@ -579,3 +582,31 @@ def test_generate_create_empty_kudu_table():
   `vrfcn_city_lon` double , PRIMARY KEY (acct_client)
 )   STORED AS kudu TBLPROPERTIES("transactional" = "false")
 ;''' in sql, sql)
+
+
+def test_create_ddl_with_abfs():
+  try:
+    from beeswax.server import dbms
+    from azure.conf import ABFS_CLUSTERS
+  except ImportError as e:
+    raise SkipTest('Hive and HiveServer2 interfaces are not enabled')
```

Comment:
This should not be needed in latest Hue: beeswax is never blacklisted, as well as azure which is a desktop lib

Reply:
Will remove